.github/workflows/release-please.yml

```
@@ -0,0 +1,14 @@
+ on:
+   push:
+     branches:
+       - main
+ name: Push version-update PR
+ jobs:
+   release-please:
+     runs-on: ubuntu-latest
+     steps:
+       - uses: GoogleCloudPlatform/release-please-action@v2
+         with:
+           token: ${{ secrets.GITHUB_TOKEN }}
+           release-type: simple
+           package-name: release-please-action
```

```suggestion
```suggestion
name: Release Please
on:
   push:
      branches: [main]

 jobs:
   release-please:
     runs-on: ubuntu-latest

     steps:
       - uses: GoogleCloudPlatform/release-please-action@v2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           release-type: python
           package-name: rtd-bin-server
```
```